backend/director/app/models.py

```diff
@@ -9,12 +10,13 @@ class Song(models.Model):
     album_art = models.URLField(blank=True)
 
     def __str__(self):
-        return f"{self.name} by {self.artist} (id: {id})"
+        return f"{self.name} by {self.artist} (id: {self.song_id})"
 
 class User(models.Model):
     # django assigns a primary key id by default
     name = models.CharField(max_length=50)
     join_time = models.DateTimeField(auto_now_add=True)
+    # party = models.ForeignKey(Party, on_delete=models.CASCADE)
```

Comment:
Could we get rid of the commented code pls?

Reply:
removed in merge conflict resolution